.travis.yml

```diff
@@ -34,8 +34,8 @@ script:
   - export GOOS=$TRAVIS_GOOS
   - export CGO_ENABLED=$TRAVIS_CGO_ENABLED
   - DCO_VERBOSITY=-q script/validate/dco
-  - GOOS=linux script/setup/install-dev-tools
-  - script/validate/vendor
+  - GOOS=linux GO111MODULE=off script/setup/install-dev-tools
+  - GO111MODULE=on go mod verify
```

Comment:
@tariq1890 is this sufficient to replace the validator script we had before? Running this locally I've manually removed some of the contents of `vendor/` and I still get

```
$ GO111MODULE=on go mod verify
all modules verified
```

Reply:
Looks like `go mod verify` may actually update dependencies? https://github.com/golang/go/issues/31372

Reply:
It actually isn't. I'll fix this